src/AppRoutes.tsx

```diff
@@ -60,8 +61,8 @@ import { VariablesPage } from 'components/VariablesPage/VariablesPage';
 
 import { AUTH_ROLE } from 'services/keycloakService';
 
-export const AppRoutes = () => (
-  <Routes>
+export const AppRoutes = createRoutesFromElements(
```

Comment:
I would rather apply the function where the `createBrowserRouter` is called.
This way, AppRoutes remains a component.

Reply:
sure :+1: